Movie_MVP/MovieMVPTests/BuilderTest.swift

```diff
@@ -0,0 +1,46 @@
+// BuilderTest.swift
+// Copyright © RoadMap. All rights reserved.
+
+@testable import Movie_MVP
+import XCTest
+
+/// Тест сборщика модулей
+final class BuilderTest: XCTestCase {
```

Comment:
```suggestion
final class BuilderTests: XCTestCase {
```
где то Tests а где-то Test
наверное стоит везде одинаково сделать Tests